src/widget/input/completion.rs

```diff
@@ -354,3 +281,193 @@ impl fmt::Display for Arg {
         }
     }
 }
+
+#[derive(Debug, Clone, Default)]
+struct Users {
+    prompt: String,
+    filtered: Vec<String>,
+    selected: Option<usize>,
+}
+
+impl Users {
+    fn process(&mut self, input: &str, users: &[User]) {
+        let (_, rest) = input.rsplit_once(' ').unwrap_or(("", input));
+
+        if rest.is_empty() {
+            *self = Self::default();
+            return;
+        }
+
+        self.selected = None;
+        self.prompt = rest.to_string();
+        self.filtered = users
+            .iter()
+            .filter_map(|user| {
+                let nickname = user.nickname();
+                nickname
```

Comment:
A tiny thing could be that we lower-cased both `rest` and `nickname` so we can tab-complete from `t` to `Tarkah`.

Not blocking but just an idea.

Reply:
Good idea! 09c9e78